pkg/search/options.go

```diff
@@ -464,28 +465,34 @@ var (
 	TestNestedStringCount       = newDerivedFieldLabel("Test Nested String Count", TestNestedString, CountDerivationType)
 	TestNestedString2Count      = newDerivedFieldLabel("Test Nested String 2 Count", TestNestedString2, CountDerivationType)
 	TestParent1StringSliceCount = newDerivedFieldLabel("Test Parent1 String Slice Count", TestParent1StringSlice, CountDerivationType)
+	TestEnum1Custom             = newDerivedFieldLabelWithType("Test String Affected By Enum1", TestEnum, CustomFieldType, postgres.Integer)
```

Comment:
I also worry that would be a larger change and take a while to get correct.  As in I worry if it would delay us getting this solved for 4.6 and at the same time reducing our capacity concerning our other priorities.

Reply:
Yeah I agree it will be a bigger and more complex change which we can do later when we have more time.